lib/robots/dor_repo/gis_assembly/extract_boundingbox.rb

```diff
@@ -15,55 +15,38 @@ def initialize
         def perform_work
           logger.debug "extract-boundingbox working on #{bare_druid}"
 
-          raise "extract-boundingbox: #{bare_druid} cannot locate normalized data: #{zip_filename}" unless File.size?(zip_filename)
-
-          extract_data_from_zip
-          raise "extract-boundingbox: #{bare_druid} cannot locate #{tmpdir}" unless File.directory?(tmpdir)
-
-          begin
+          normalizer.with_normalized do |tmpdir|
```

Comment:
Extract Bounding Box robot now uses a normalizer to create a temp normalized data files instead of extracting from zip.